### PR TITLE
[5.3] Add support for Gammu SMS Api

### PR DESCRIPTION
**Pull request by kristiandrucker**
Added support for [Gammu SMS Api](https://github.com/kristiandrucker/gammuApi). Tested it with latest Laravel 5.3 rc1.

### CODE REVIEW

**Comment by GrahamCampbell** [comment]
Thanks, but I don't think we're accepting new drivers here.

**Comment by kristiandrucker** [comment]
Why? Its good one to have it. Someone would like to use their own gammu sms system with Laravel, so we could allow it.

**Comment by GrahamCampbell** [comment]
Yeh, but someone would like to use 100 other things too if you see what I mean.

**Comment by kristiandrucker** [comment]
Yes I see.

File: src/Illuminate/Notifications/Channels/SMSChannel.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+	 * Send the given notification.
+	 *
+	 * @param  mixed  $notifiable
+	 * @param  \Illuminate\Notifications\Notification  $notification
+	 * @return void
+	 */
+	public function send($notifiable, Notification $notification)
+	{
+		$message = $notification->toSMS($notifiable);
+		$this->http->post(env('SMS_URL') . '/send/sms', [
```
Comment (GrahamCampbell) [inline src/Illuminate/Notifications/Channels/SMSChannel.php]: The env function is not available for use outside on config files.

Comment (kristiandrucker) [inline src/Illuminate/Notifications/Channels/SMSChannel.php]: So should I pass it as a variable into construct?

**Comment by kristiandrucker** [comment]
Ok changed it to get key and url from config.

**Comment by themsaid** [comment]
@kristiandrucker you can submit your channel to this GitHub Organisation if you want:
https://github.com/laravel-notification-channels

We created it to house all custom channels that aren't built-in.

**Comment by kristiandrucker** [comment]
@themsaid Thanks I will look into it.

**Comment by taylorotwell** [comment]
Yes, please use the community project to add more drivers.

**Pull request closed without merging**